DOCS-7058 administer community (#128)
mulesoft/docs-api-community-manager: modules/ROOT/pages/administer-community.adoc

= Administer Your API Community
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> Administering your Anypoint API Community Manager community includes staying up to date with important notifications such as new releases and downtime, upgrading API Community Manager, and getting support from MuleSoft.

Reviewer: Possible alternative:
```suggestion
Administering your Anypoint API Community Manager community requires monitoring important notifications such as new releases and downtime, upgrading API Community Manager, and getting support from MuleSoft.
```

Reviewer 2: agree
